src/utils/floating-ui.ts

```diff
@@ -380,13 +433,21 @@ export function connectFloatingUI(
 
   disconnectFloatingUI(component, referenceEl, floatingEl);
 
+  // ensure position matches for initial positioning
+  floatingEl.style.position = component.overlayPositioning;
+
+  const runAutoUpdate = Build.isBrowser
+    ? autoUpdate
+    : (_refEl: HTMLElement, _floatingEl: HTMLElement, updateCallback: Function): (() => void) => {
+        updateCallback();
+        return () => {
+          /* noop */
+        };
+      };
+
   cleanupMap.set(
     component,
-    autoUpdate(referenceEl, floatingEl, () => {
-      if (component.open) {
```

Comment:
No more open check right? I don't think we need the open check anymore.

Reply:
The `reposition` util takes care of it now, so no need for it here.